### PR TITLE
docs: fix typos and minor grammatical issues

### PR DESCRIPTION
**Pull request by hugo-gomes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,10 +180,10 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
...
+| [Reviewpad](https://www.reviewpad.com) [:rocket:](https://reviewpad.com/discord/hiring) [:octocat:](https://github.com/reviewpad) | Pull request automation to safely merge faster with happy developers. | `Porto` <br> `Remote` |
 | [Rows](https://www.rows.com) [:rocket:](https://github.com/dashdash/hiring) | Create interactive web apps using spreadsheet skills. | `Porto` |
 | [SingleStore](https://www.singlestore.com/) [:rocket:](https://www.singlestore.com/careers/jobs/) | The scalable cloud database for data-intensive applications, deployed on AWS, Azure, or GCP. (formerly MemSQL) | `Lisboa` `Remote` |
-| [Synopsys](https://www.synopsys.com) [:rocket:](https://sjobs.brassring.com/TGnewUI/Search/Home/Home?partnerid=25235&siteid=5359) | Silicon chip design, verification, IP integration and application security | `Lisbon` `Porto` |
+| [Synopsys](https://www.synopsys.com) [:rocket:](https://sjobs.brassring.com/TGnewUI/Search/Home/Home?partnerid=25235&siteid=5359) | Silicon chip design, verification, IP integration and application security | `Liaboa` `Porto` |
```
Comment (afonsojramos) [inline README.md:186]:
```suggestion
| [Synopsys](https://www.synopsys.com) [:rocket:](https://sjobs.brassring.com/TGnewUI/Search/Home/Home?partnerid=25235&siteid=5359) | Silicon chip design, verification, IP integration and application security | `Lisboa` `Porto` |
```

File: README.md [hunk 2]
```diff
@@ -268,15 +268,15 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
-| [Alviere](https://alviere.com/) [:rocket:](https://alviere.com/careers) [:octocat:](https://github.com/Alviere) | Full-Stack embedded finance plaftorm. | `Lisbon` |
+| [Alviere](https://alviere.com/) [:rocket:](https://alviere.com/careers) [:octocat:](https://github.com/Alviere) | Full-Stack embedded finance plaftorm. | `Liaboa` |
```
Comment (afonsojramos) [inline README.md:271]:
```suggestion
| [Alviere](https://alviere.com/) [:rocket:](https://alviere.com/careers) [:octocat:](https://github.com/Alviere) | Full-Stack embedded finance plaftorm. | `Lisboa` |
```

File: README.md [hunk 3]
```diff
@@ -268,15 +268,15 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
...
 | [Anchorage](https://anchorage.com/) [:rocket:](https://anchorage.com/careers/) | The most advanced digital asset platform for secure crypto custody. | `Porto` `Remote` |
-| [Banco Atlântico Europa](https://www.atlantico.eu/) [:rocket:](https://www.atlantico.eu/institucional/carreiras) | An European bank, under the supervision of Banco de Portugal. | `Lisboa` |
+| [Banco Atlântico Europa](https://www.atlantico.eu/) [:rocket:](https://www.atlantico.eu/institucional/carreiras) | A European bank, under the supervision of Banco de Portugal. | `Lisboa` |
```
Comment (afonsojramos) [inline README.md:273]: Had to look this one up, but you are correct!

**Review by afonsojramos — CHANGES_REQUESTED** [review]
Looks good, just a couple of typos 👀

File: README.md (hunk 1 above)
Comment (hugo-gomes) [reply README.md]: 😅

File: README.md (hunk 2 above)
Comment (hugo-gomes) [reply README.md]: 😅

File: README.md (hunk 3 above)
Comment (hugo-gomes) [reply README.md:273]: Yep, "it's all about the sound" 🙂

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**